BrainPortal/app/models/cluster_task.rb

```diff
@@ -2557,7 +2562,9 @@ def local_dp_storage_paths #:nodoc:
 
   # Just invokes the same method on the task's ToolConfig.
   def ext3capture_basenames
-    self.tool_config.ext3capture_basenames
+    names = self.tool_config.ext3capture_basenames
+    self.addlog("Overlaying ext3 capturing basenames #{names}")
```

Comment:
Don't addlog() in a method that could be called just to fetch some information. Addlog when the information is used instead (so at the place where the method is called).

Reply:
E.g. when calling the method in the console.